usr/src/uts/common/os/kmem.c

```diff
@@ -2186,12 +2186,10 @@ kmem_depot_ws_zero(kmem_cache_t *cp)
 }
 
 /*
- * Setting it this high (1GB) causes the explicit kpreempt() to have very
- * little effect. To make calls to kpreempt() more frequent, use a smaller
- * value. A few megabytes is usually small enough that it will run hundreds of
- * times per second.
+ * Setting this to 1MB causes us to call kpreempt() approximately
+ * once per 5-500 milliseconds.
```

Comment:
5-500 is a pretty large range.  I feel like if we're going to be this vague, then it's better to use more natural language (instead of using the rather scientific looking ranges).  So, personally I prefer "hundreds of times per second" since that communicates the idea that it's "often".

Another thought about this comment: the old comment gave me two datapoints which allowed me to extrapolate what larger values do vs. smaller ones.  The new comment gives me a single data point.  I haven't looked at the rest of the change, so for all I know this is described in a big comment somewhere else.